src/session-list.vala

```diff
@@ -53,7 +47,9 @@ public class SessionPrompt : PromptBox
         }
         else
         {
-            foreach (var session in sessions_sorted_ci( LightDM.get_sessions() ) )
+            var sessions = LightDM.get_sessions().copy();
+            sessions.sort_with_data((a, b) => GLib.strcmp (a.name.casefold(), b.name.casefold()));
```

Comment:
Great bugfix, but if we're at it anyway, maybe we want to use

```vala
            sessions.sort_with_data((a, b) => GLib.strcmp (a.name.casefold().collate_key(), b.name.casefold().collate_key()));
```

instead, making sure that we sort using the [linguistically (almost) correct rules for the current locale](https://valadoc.org/glib-2.0/string.collate.html)?